x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart.js

```diff
@@ -1199,19 +1199,20 @@ class TimeseriesChartIntl extends Component {
 
     contextGroup
       .append('g')
-      .attr('class', 'x brush')
+      .attr('class', 'x brush smvTimeSlider')
```

Comment:
Is this `smvTimeSlider` class needed?

Reply:
Removed here https://github.com/elastic/kibana/pull/117946/commits/23d64d3b13b3ce71fe84a579719fb5210eef48f5